Towards version 0.3.0 (#71)
xlmnxp/blue-recorder-rs: README.md

- You can choose the audio input source you want from the list.
- You can also set the default values you want by simply changing them in the interface, and the program will save them for you for the next time you open it. 

> It based on [Green Recorder](https://github.com/mhsabbagh/green-recorder) and rewritten in Rust with improvements.

Reviewer: ```suggestion
This project is based on [Green Recorder](https://github.com/mhsabbagh/green-recorder) and rewritten in Rust with improvements.
```

> It based on [Green Recorder](https://github.com/mhsabbagh/green-recorder) and rewritten in Rust with improvements.

Reviewer: ```suggestion
This project is based on [Green Recorder](https://github.com/mhsabbagh/green-recorder) and is rewritten in Rust with improvements.
```

## Installation

> Blue Recorder available as Snap and Flatpak package:

Reviewer: ```suggestion
Blue Recorder is available as a Snap and Flatpak package.
```

<a href='https://snapcraft.io/blue-recorder'><img height='60' alt='Get it from the Snap Store' src='https://snapcraft.io/static/images/badges/en/snap-store-black.svg'/></a> <a href='https://flathub.org/apps/details/sa.sy.bluerecorder'><img height='60' alt='Download on Flathub' src='https://flathub.org/assets/badges/flathub-badge-en.png'/></a>

## Build from source

> Blue Recorder depend in the following dependencies:

Reviewer: ```suggestion
Blue Recorder depends on the following dependencies:
```

```
ffmpeg
gtk
gdk
gio
x11-utils
```

> Install dependencies Ubuntu and Debian based distros:

Reviewer: ```suggestion
Install dependencies for Ubuntu and Debian-based distros:
```
